phase1/AbdulWahab/Java/Schema/src/main/java/com/schema_database_validator/Schema/validator/CustomJsonSchemaValidator.java

```diff
@@ -0,0 +1,87 @@
+package com.schema_database_validator.schema.validator;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.context.annotation.Configuration;
+
+
+@Configuration
+public class CustomJsonSchemaValidator {
```

Comment:
what will happen if we have a key and value is object? 
where are you validation nested objects?

Reply:
where are you validating objects in array?